_posts/en/newsletters/2021-03-17-newsletter.md

```diff
@@ -0,0 +1,151 @@
+    the LN specification to help facilitate funding recovery efforts.
+    As LN software adds support for funding channels from external
+    wallets (e.g. C-Lightning as described in [Newsletter #51][news51
+    cl2672] and LND in [Newsletter #92][news92 lnd4079]), developers may
+    want to give these sort of failure scenarios more attention.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 2.0.0RC2][hwi 2.0.0] is the release for the next major
+  version of HWI.  Among other improvements, it contains support for
+  multisig on the BitBox02, improved documentation, and support for
+  paying `OP_RETURN` outputs with a Trezor.
+
+- [Rust-Lightning 0.0.13][] is the latest release for this LN library
+  containing improvements aimed at forward compatibility with [multipath
+  payments][topic multipath payments] and future script upgrades such as
+  [taproot][topic taproot].
+
+- [BTCPay Server 1.0.7.0][] is the latest release for this self-hosted
+  payment processing software.  Notable improvements include a more
+  featureful and visually appealing wallet setup wizard, the ability to
+  use a Specter hardware wallet, and more efficient QR codes for [bech32
+  addresses][topic bech32].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21007][] adds a new `-daemonwait` configuration option. It has
+  been possible to run Bitcoin Core as a background daemon process since early
+  versions by starting the program with the `-daemon` configuration option.
+  The `-daemon` option causes the program to immediately start the daemon process
+  in the background. The new `-daemonwait` option
+  is similar, but only starts the background daemon process after initialization
+  is complete. This allows the user or parent process to more easily know
+  whether the daemon started successfully by observing the program's output or
+  exit code.
+
+- [C-Lightning #4404][] allows the `keysend` RPC (see [Newsletter
+  #107][news107 keysend]) to send messages even to nodes that don't
+  explicitly signal that they support the feature.  As
+  [discussed][fromknecht keysend], the signal was never standardized and
+  the procedure implemented by LND didn't depend on signaling, so this
+  change should allow C-Lightning to send to roughly the same set of
+  nodes that LND can address.
+
+- [C-Lightning #4410][] brings the experimental implementation for
```

Comment:
Do we have enough news items for a dual funding topic?

Reply:
Yes and it's a topic I'd like to write.